edk2toolext/edk2_invocable.py

```diff
@@ -12,16 +12,35 @@
 import sys
 import logging
 import inspect
+import pkg_resources
 import argparse
 from edk2toolext.environment import shell_environment
 from edk2toollib.utility_functions import GetHostInfo
+from edk2toolext.environment import version_aggregator
 from edk2toollib.utility_functions import locate_class_in_module
 from edk2toollib.utility_functions import import_module_by_file_name
 from edk2toolext.base_abstract_invocable import BaseAbstractInvocable
 
 
 class Edk2Invocable(BaseAbstractInvocable):
 
+    #
+    # Pass in a list of pip package names and they will be printed as well as
+    # reported to the global version_aggregator
+    @classmethod
+    def _collect_python_pip_info(cls):
```

Comment:
I don't think a class function should start with _ since that is an indicator of private and by definition a class function is not private

Reply:
Fair